examples/pytorch.ipynb

```diff
@@ -14,13 +18,19 @@
     "from torch import optim\n",
     "from torch.utils.data import TensorDataset, DataLoader\n",
     "\n",
-    "from livelossplot import PlotLosses"
+    "from livelossplot.plot_losses import PlotLosses\n",
+    "from livelossplot.output_plugins.extrema_print import ExtremaPrint\n",
+    "from livelossplot.output_plugins.matplotlib import Matplotlib"
```

Comment:
We should still be able to do just `from livelossplot import PlotLosses`.
Otherwise, it soon turns into Java.

Reply:
done